<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>571</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string>What would you like to do?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="learnBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Learn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="practiceBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Practice</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quizBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>351</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Take a Quiz</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
